app/views/flood_risk_engine/enrollments/addresses/_form.html.erb

```diff
@@ -18,39 +21,34 @@
     )
   %>
 </p>
-
-<!-- Manually entered address -->
-<fieldset>
-  <legend class="visuallyhidden"><%= t(".legend") %></legend>
-
-  <%= form_group_and_validation(form, :premises) do %>
-    <%= f.label :premises, class: "form-label" do %>
-      <%= t(".premises.label") %>
-      <span class="form-hint"><%= t(".premises.hint") %></span>
-    <% end %>
-    <%= f.text_field :premises, class: "form-control form-control-char-50" %>
-  <% end %>
-
-  <%= form_group_and_validation(form, :street_address) do %>
-    <%= f.label :street_address, t(".street_address"), class: "form-label" %>
-    <%= f.text_field :street_address, class: "form-control form-control-char-50" %>
-  <% end %>
-
-  <%= form_group_and_validation(form, :locality) do %>
-    <%= f.label :locality, t(".locality"), class: "form-label" %>
-    <%= f.text_field :locality, class: "form-control form-control-char-50" %>
-  <% end %>
-
-  <%= form_group_and_validation(form, :city) do %>
-    <%= f.label :city, t(".city"), class: "form-label" %>
-    <%= f.text_field :city, class: "form-control form-control-char-35" %>
-  <% end %>
-</fieldset>
-
-<%= hidden_field_tag :step_back_to_postcode, params.fetch(:step_back_to_postcode, false) %>
-<%= hidden_field_tag :target_step,  params.fetch(:target_step, nil) %>
```

Comment:
Do we need to hang onto these two fields for now, at least until the new workflow stuff is merged in?

Reply:
Those lines shouldn't have been removed. Will restore them.